[2024-05-06T03:11:01.178+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T03:10:24.544407+00:00 [queued]>
[2024-05-06T03:11:01.195+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T03:10:24.544407+00:00 [queued]>
[2024-05-06T03:11:01.196+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-05-06T03:11:01.215+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): write_to_clean_data_task> on 2024-05-06 03:10:24.544407+00:00
[2024-05-06T03:11:01.231+0000] {standard_task_runner.py:57} INFO - Started process 116 to run task
[2024-05-06T03:11:01.239+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main', 'write_to_clean_data_task', 'manual__2024-05-06T03:10:24.544407+00:00', '--job-id', '204', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpg3zuz2md']
[2024-05-06T03:11:01.246+0000] {standard_task_runner.py:85} INFO - Job 204: Subtask write_to_clean_data_task
[2024-05-06T03:11:01.339+0000] {task_command.py:415} INFO - Running <TaskInstance: main.write_to_clean_data_task manual__2024-05-06T03:10:24.544407+00:00 [running]> on host 172d1424854d
[2024-05-06T03:11:01.478+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='write_to_clean_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T03:10:24.544407+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T03:10:24.544407+00:00'
[2024-05-06T03:11:01.625+0000] {warnings.py:109} WARNING - /opt/***/dags/main.py:72: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2024-05-06T03:11:22.155+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-06T03:11:22.175+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main, task_id=write_to_clean_data_task, execution_date=20240506T031024, start_date=20240506T031101, end_date=20240506T031122
[2024-05-06T03:11:22.270+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-06T03:11:22.310+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
